api/v1beta1/openstackdataplaneservice_types.go

```diff
@@ -76,6 +76,17 @@ type OpenStackDataPlaneServiceSpec struct {
 	// OpenStackAnsibleEERunnerImage image to use as the ansibleEE runner image
 	// +kubebuilder:validation:Optional
 	OpenStackAnsibleEERunnerImage string `json:"openStackAnsibleEERunnerImage,omitempty" yaml:"openStackAnsibleEERunnerImage,omitempty"`
+
+	// AnsibleMaxFailPercentage is used to tune service specific, allowable failure percentages during the Ansible execution
+	// https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_error_handling.html#setting-a-maximum-failure-percentage
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Minimum=1
+	// +kubebuilder:validation:Maximum=100
+	AnsibleMaxFailPercentage int `json:"ansibleMaxFailPercentage,omitempty"`
```

Comment:
You mentioned safe defaults in the commit message but I don't see those defaults here. Where are they set?

Reply:
Yeah, I think I need to update my wording there. We'll just fallback to what Ansible would do without these arguments provided.